### DISCUSSION TITLE
Problem with hybrid runs

### DISCUSSION THREAD

**Discussion by adamrher** [NCAR/amwg_dev#459] [Discussion about coupled runs]
Hybrid runs 64(b,c) have very large global SSTs that suggest they are not being initialized to the year 43 SST state. @cecilehannay relayed to me that @gustavo-marques verified that these runs are reading in the year 43 restart file. If so, I still suspect it's not being used. Runs 64(e,f) are branch runs but w/ small perturbation to clubb gamma, and those look like there are initialized correctly.

![temp_tseries](https://github.com/NCAR/amwg_dev/assets/29961476/4ffc532f-956c-4129-be5f-4c5347023c12)

We need to be able to do hybrid runs, and so this issue needs to be resolved.

**Comment by gustavo-marques** [comment]
I was looking at the wrong log file previously.
Cecile and I looked again at 64b this afternoon and we indeed see it was not initialized from the desired restart file.  
We will look into fixing this issue asap.

**Reply by adamrher** [reply]
great thx Gustavo.

**Comment by gustavo-marques** [comment]
Here is a short-term hack for fixing this:

1) Add the following into `user_nl_mom` (MOM6 restart filename must be changed accordingly)

```
INPUTDIR = "INPUT"
INIT_LAYERS_FROM_Z_FILE = False
THICKNESS_CONFIG = "thickness_file"
THICKNESS_FILE = "b.e23_alpha16g.BLT1850.ne30_t232.064.mom6.r.0043-01-01-00000.nc"
TS_CONFIG = "file"
TS_FILE = "b.e23_alpha16g.BLT1850.ne30_t232.064.mom6.r.0043-01-01-00000.nc"
TEMP_IC_VAR = "Temp"
SALT_IC_VAR = "Salt"
VELOCITY_CONFIG = "file"
VELOCITY_FILE = "b.e23_alpha16g.BLT1850.ne30_t232.064.mom6.r.0043-01-01-00000.nc"
```

2) Go to the run directory and do the following

```
mkdir INPUT
cd INPUT
ln -s /glade/campaign/cesm/cesmdata/inputdata/ocn/mom/tx2_3v2/* .
ln -s /glade/campaign/cesm/cesmdata/inputdata/cesm2_init/b.e23_alpha16g.BLT1850.ne30_t232.064/0043-01-01/b.e23_alpha16g.BLT1850.ne30_t232.064.mom6.r.0043-01-01-00000.nc .  # or the MOM6 restart file that must be used
```

For reference, here is a version of run 064b with this hack applied:

`/glade/derecho/scratch/gmarques/b.e23_alpha16g.BLT1850.ne30_t232.064b.clone`

**Reply by jedwards4b** [reply]
Would you like to brainstorm on how to fix this or do you have a plan?

**Comment by gustavo-marques** [comment]
I would like to brainstorm. Thanks, Jim! I will send you a meeting invite.
